pkgs/by-name/sy/sync_music/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, ffmpeg
+, mutagen
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "sync_music";
+  version = "0.5.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "fetzerch";
+    repo = "sync_music";
+    rev = "v${version}";
+    hash = "sha256-8vDSTaTQvdADTSxbfSxgCDsFbGPD5RtQGURv3LTwGXQ=";
+  };
+
+  propagatedBuildInputs = [
+    python3Packages.mutagen
+    python3Packages.pbr
+    ffmpeg
```

Related to the `pyproject` recommendation above, to make it build with setuptools.
```suggestion
    python3Packages.setuptools
    ffmpeg
```